salt/modules/htpasswd.py

```diff
@@ -139,3 +139,46 @@ def userdel(pwfile, user, runas=None):
     out = __salt__['cmd.run'](cmd, runas=runas,
                               python_shell=False).splitlines()
     return out
+
+
+def verify(pwfile, user, password, opts='', runas=None):
+    '''
+    Return True if the htpasswd file exists, the user has an entry, and their
+    password matches.
+
+    pwfile
+        Path to htpasswd file
+
+    user
+        User name
+
+    password
+        User password
+
+    opts
+        Valid options that can be passed are:
```

Comment:
I modeled it after the other two methods in this module, as well as the (currently 1, but soon 2 - I have another PR that I'll put out once this one is merged!) functions in `salt/states/htpasswd.py` which allow the user to pass in options, which get passed through to these module fns.

In fact, if I were to remove this argument and change to arg-per-option, then I'd have to duplicate that logic in the `webutil.user_exists` state, so it could unpack from cmdline flags to fn-arguments, just so this module fn could undo that work and put them back into cmdline flags. Plus that unpack/repack cycle would be pretty brittle - changes in one place have to be synchronized with the other, otherwise things break in subtle, non-obvious, no-error-throwing ways.